languagetool-language-modules/sr/src/main/java/org/languagetool/language/BosniaSerbian.java

```diff
@@ -0,0 +1,15 @@
+package org.languagetool.language;
+
+public class BosniaSerbian extends Serbian {
```

Comment:
Could you add a `@since 4.0` javadoc comment to all new Java classes?

Reply:
Yes, will add.